tests/test_common_view_logic.py

```diff
@@ -1,389 +1,182 @@
 import unittest
+import vcr
 
-from unittest.mock import patch
+from canonicalwebteam.blog import wordpress_api as api
 from canonicalwebteam.blog.common_view_logic import (
+    get_complete_article,
     get_index_context,
+    get_group_page_context,
+    get_topic_page_context,
     get_article_context,
 )
+from canonicalwebteam.http import CachedSession
+
+# make sure that timeouts do not hinder tests
+api.api_session = CachedSession(fallback_cache_duration=3600, timeout=(1, 100))
```

Comment:
Remove cache for tests

`api.api_session = Session()`

Reply:
This way the tests wont reflect the actual setup though. Id rather keep if for now. There shouldnt be any harm, should there?

If the cache misbehaves, the tests should fail :)

Reply:
well then you are not reflecting the actual behaviour if you change the object. If you want to keep it then keep the Session as it is.

Reply:
Alright, Ill remove the timeout tolerance increase here to reflect the actual problems of the API.

Reply:
Nvm, using `CachedSession` does not work without the timeout increase.